### DISCUSSION TITLE
Show command outputs

### DISCUSSION THREAD

**Discussion by truh** [pester/Pester#2406] [Q&A] [answered]
How can get Pester not to hide the outputs of the commands that I run in the test cases.

**Comment by fflaten** [comment, marked as answer]
You can't atm, but you can either:
- Pass it to something like `Out-String | Write-Host` (ugly)
- Access the output through the `StandardOutput` property in the result-object:
  ```powershell
  (Invoke-Pester -PassThru).Tests.StandardOutput
  ```

**Comment by nohwnd** [comment]
What is your motivation for such feature? Do you use it for debugging? Is it useful to have the output "line-by-line" or it is enough to write it all at once once the test finishes?


If you were to implement this, I assume you would need to add a pipeline [here](https://github.com/pester/Pester/blob/main/src/Pester.Runtime.ps1#L1619), that would write to console using Write-Host, and then forward the input to the StandardOutput variable. 

And add an option for that into pester configuration.
